static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -141,7 +199,64 @@ class Issues extends Component<Props, State> {
     }
   }
 
+  async fetchIssuesCount() {
+    const {api, version} = this.props;
+    const endpoint = this.getIssueCountEndpoint();
+
+    try {
+      const response = await api.requestPromise(endpoint);
+      this.setState({
+        count: {
+          release: response[`release:${version}`],
+          firstRelease: response[`first-release:${version}`],
+          resolved: response['is:resolved'],
+          unhandled: response['error.handled:0'],
```

Comment:
Should we store these queries in the constant/enum so that we can reuse the in both `fetchIssuesCount` and `getIssueCountEndpoint`?